sdk/eventhub/.openpublishing.publish.config.json

```diff
@@ -2,7 +2,7 @@
   "docsets_to_publish": [
     {
       "docset_name": "event-hubs-js",
-      "build_source_folder": "client/docs",
+      "build_source_folder": "event-hubs/docs",
       "build_output_subfolder": "event-hubs-js",
```

Comment:
The `docs` directory is not moved either (it looks to only contain some generic files.) I know little about docs.  @ramya-rao-a do you know whether the `docs` directory is needed?

Reply:
That is an artifact from the automated API reference generation.
Now that we are in the main repo, we shouldn't need this. 
Logged #1972 to track its removal

Reply:
Thanks!!